web/src/styles/base/typography.scss

```diff
@@ -1,7 +1,7 @@
 @font-face {
   font-family: "Public Sans Web";
   font-style: normal;
-  font-weight: 600;
+  font-weight: 500;
```

Comment:
[boulder] 600 was correct. [Public Sans Web Semi-Bold is a font-weight of 600](https://fonts.google.com/specimen/Public+Sans).

Reply:
I misunderstood the comments, thank you, confirmed with Kim as well